doc/manual/embedding.rst

```diff
@@ -195,9 +195,9 @@ Manipulating the Garbage Collector
 There are some functions to control the GC. In normal use cases, these should not be necessary.
 
 ========================= ==============================================================================
-``void jl_gc_collect()``   Force a GC run
-``void jl_gc_disable()``   Disable the GC
-``void jl_gc_enable()``    Enable the GC
+``void jl_gc_collect()``  Force a GC run
```

Comment:
as long as the table looks right in the html-rendered version then should be fine